swagger_spec_validator/validator20.py

```diff
@@ -35,10 +36,14 @@ def validate_spec(spec_json, _spec_url=None):
     """
     validate_json(spec_json, 'schemas/v2.0/schema.json')
 
+    # Dereference all $refs so we don't have to deal with them
+    fix_malformed_model_refs(spec_json)
```

Comment:
Like the [TODO1](https://github.com/Yelp/bravado-core/blob/master/bravado_core/spec.py#L201) and [TODO2](https://github.com/Yelp/bravado-core/blob/master/bravado_core/model.py#L173) say in bravado-core, this would ideally be upstreamed and live in [jsonref](https://github.com/gazpachoking/jsonref) for all to share.  Short term, I find this acceptable given the circumstances.